bigchaindb/__init__.py

```diff
@@ -62,7 +62,6 @@
         'loglevel': logging.getLevelName(
             log_config['handlers']['console']['level']).lower(),
         'workers': None,  # if none, the value will be cpu_count * 2 + 1
-        'threads': None,  # if none, the value will be cpu_count * 2 + 1
```

Comment:
Actually I didn't; I just hid it. If you set the parameter it'll still have effect.